### ISSUE TITLE
inspec compliance login fails

### ISSUE THREAD

**Issue by mhedgpeth** [inspec/inspec#689]
### Description

I am unable to log in with inspec compliance login
### InSpec and Platform Version

0.19.3 (latest)
### Replication Case

Run the following command:

```
inspec compliance login https://myurl.centralus.cloudapp.azure.com --user myuser --password mypassword
```
### Possible Solutions

I wonder if this has something to do with the server https not working. I tried adding the --insecure flag and got the same results.
### Stacktrace

Please include the stacktrace output or link to a gist of it, if there is one.

```
C:/Users/mh185122/AppData/Local/chefdk/gem/ruby/2.1.0/gems/inspec-0.19.3/lib/bundles/inspec-compliance/api.rb:21:in `leg
acy_login_post': undefined method `send_request' for Compliance::API:Class (NoMethodError)
        from C:/Users/mh185122/AppData/Local/chefdk/gem/ruby/2.1.0/gems/inspec-0.19.3/lib/bundles/inspec-compliance/cli.
rb:206:in `login_legacy'
        from C:/Users/mh185122/AppData/Local/chefdk/gem/ruby/2.1.0/gems/inspec-0.19.3/lib/bundles/inspec-compliance/cli.
rb:36:in `login'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_comma
nd'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/invocation.rb:115:in `invoke'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor.rb:235:in `block in subcommand'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_comma
nd'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
        from C:/Users/mh185122/AppData/Local/chefdk/gem/ruby/2.1.0/gems/inspec-0.19.3/bin/inspec:9:in `<top (required)>'

        from c:/Users/mh185122/AppData/Local/chefdk/gem/ruby/2.1.0/bin/inspec:22:in `load'
        from c:/Users/mh185122/AppData/Local/chefdk/gem/ruby/2.1.0/bin/inspec:22:in `<main>'
```

**Comment by srenatus** [comment]
Which version of compliance do you use? While this looks like a real bug in the backwards-compatibility of inspec-compliance, using the old auth way hints at a not-quite-recent Compliance version...

**Comment by mhedgpeth** [comment]
I downloaded it and loaded it last night. The auth method could be my
ignorance; where can I read more about the new auth?

**Comment by chris-rock** [comment]
@mhedgpeth I know we have some bugs there. I am already in the process of fixing the bug...

**Comment by mhedgpeth** [comment]
Fantastic, I'm looking forward to seeing it working :+1:

**Comment by chris-rock** [comment]
fixed in #695

**Comment by mhedgpeth** [comment]
Verified that this is fixed.
